### PR TITLE
[Exercise 10]: Setup an observability stack

### PR DESCRIPTION
**Pull request by aashirathore6362**

### CODE REVIEW

File: .github/workflows/std-workflows.yml [hunk 1]
```diff
@@ -7,6 +7,9 @@ on:
     paths-ignore:
      - 'helmcharts/**'
      - '.github/**'
+     - 'blackbox-exporter-values.yaml'
```
Comment (jdk2588) [inline .github/workflows/std-workflows.yml:10]: Reason to ignore these paths?

File: makefile [hunk 2]
```diff
@@ -2,7 +2,7 @@ install-dep:
 	pip install -r requirements.txt
 
 run:install-dep
-	flask -A student.py run --host=${HOSTPORT} --port=${APP_PORT}
+	flask -A student.py run --host=${HOSTPORT} --port=${APP_PORT} --debug
```
Comment (jdk2588) [inline makefile:5]: Why debug-mode?

File: runapp.sh [hunk 3]
```diff
@@ -6,5 +6,5 @@ then
     flask db upgrade
 else
     echo "Migrations not running.";
-    flask run
+    flask run --debug
```
Comment (jdk2588) [inline runapp.sh:9]: Why debug mode?

**Review by jdk2588 — CHANGES_REQUESTED** [review]

File: .github/workflows/std-workflows.yml (hunk 1 above)
Comment (aashirathore6362) [reply .github/workflows/std-workflows.yml:10]: No need to generate new docker image, this is values file for helm charts.

File: makefile (hunk 2 above)
Comment (aashirathore6362) [reply makefile:5]: For console logs.

**Pull request merged**